demisto_sdk/commands/content_graph/parsers/json_content_item.py

```diff
@@ -24,6 +23,8 @@ def __init__(
         git_sha: Optional[str] = None,
     ) -> None:
         super().__init__(path, pack_marketplaces)
+        self.set_path_with_suffix(".json")
```

Comment:
```suggestion
        self.path = self.set_path_with_suffix(".json")
```

Reply:
i don't like assigning class attributes within methods especially when they are executed from the constructors.
IMO this is more readable.

Reply:
agree, changed it now